modules/openapi-generator/src/main/resources/dart/libraries/dio/serialization/freezed/variable_type.mustache

```diff
@@ -0,0 +1,14 @@
+{{!
+Required   Nullable  Result
+true	   true	     required Type?
+true	   false	 required Type
+false	   true	     Type?
+false	   false	 Type?
+}}
+
+{{#required}}
+    {{#required}}required {{/required}}{{#isContainer}}{{baseType}}<{{#isMap}}String, {{/isMap}}{{#items}}{{>serialization/built_value/variable_type}}{{/items}}>{{/isContainer}}{{^isContainer}}{{{datatypeWithEnum}}}{{/isContainer}}{{#isNullable}}?{{/isNullable}}
```

Comment:
if this is going to depend on `serialization/built_value/variable_type`, shouldn't we move it outside the `built_value`  directory ?

Reply:
Already changed it locally by duplicating it.. 😄  will be up in the next merge.. 👍